packages/fxa-payments-server/src/components/CouponForm/index.tsx

```diff
@@ -184,12 +184,12 @@ export const CouponForm = ({
       </h4>
       {hasCoupon ? (
         <div
-          className="flex justify-between items-center"
+          className="flex gap-4 justify-between items-center"
           data-testid="coupon-hascoupon"
         >
-          <div>{promotionCode}</div>
+          <div className="break-all">{promotionCode}</div>
```

Comment:
@xlisachan While I was testing the impact of removing `ml-4` on line 192, I noticed that adding a long promo code pushed the button outside its container. Adding `break-all` allows the promo code to break over two lines (if needed) without impacting the button's placement.

Reply:
Also aligned! In this case, both work, but out of curiosity, why did you opt for `break-all` and not `break-words`?

Reply:
I was thinking of promo codes being all in one chunk (not multiple words), so either could work. If there's any chance of multiple promo codes used for the same transaction, `break-words` would probably be best though - I think I'll change that.